core/consensus/aura/src/lib.rs

```diff
@@ -628,13 +643,20 @@ pub fn import_queue<B, C, E>(
 	B: Block,
 	C: 'static + Authorities<B> + ProvideRuntimeApi + Send + Sync,
 	C::Api: BlockBuilderApi<B>,
-	DigestItemFor<B>: CompatibleDigestItem + DigestItem<AuthorityId=AuthorityId>,
+	DigestItemFor<B>: CompatibleDigestItem<P> + DigestItem<AuthorityId=AuthorityId<P>>,
 	E: 'static + ExtraVerification<B>,
+	P: Pair + Send + Sync + 'static,
```

Comment:
Some of the `Send + Sync` bound may be unnecessary. I remember we only need them for `start_aura_thread`. But I'm not really sure on this.

Reply:
I am not sure either.